docs/guide/deploy.md

```diff
@@ -112,6 +112,7 @@ npm run test && pip install -r pytest_requirements.txt && pytest tests
 ```bash
 npm run config
 ```
+Note, that `anthropic.claude-instant-v1` (Claude Instant) and `amazon.titan-embed-text-v1` (Titan Embeddings G1 - Text) need to be enabled in Bedrock before you run the command.
```

Comment:
I would not specify models here. There are the model needed for integ tests.

On line 119 below  I would update instead

`we support all models provided by Bedrock` to `we support all models provided by Bedrock that [were enabled](https://docs.aws.amazon.com/bedrock/latest/userguide/model-access.html)`

Reply:
Updated